wiki/general/tutorials/getting-started-.md

```diff
@@ -0,0 +1,141 @@
+In that tutorial we will cover how to use Launchpad to generate a WordPress plugin.
+
+### Create the project
+
+#### Generate the project using composer
+The first step in creating the project itself is by using the composer command line with the following input: `composer create-project wp-launchpad/launchpad my-plugin`
+
+Once this is completed, you should be able to see a new folder with the code for your project inside it. However, it is not yet ready to be used as it is still Launchpad default code and needs to be initialized.
+
+#### Initialize project with launchpad command
+
+As multiple projects can exist on the same WordPress website at once, it is important to containerize all of them, and for that purpose we will use a specific namespace for our code.
+
+To do so with Launchpad, we need to run the following command in the terminal:
+`bin/generator initialize`
+
+Once the command has been executed, it should ask you for your plugin name, and it will generate the necessary changes from it.
```

Comment:
Giving "debug-hooks" as an input raises a PSR-4 error. However, putting "Debug Hooks" is properly handled and generates file named "debug-hooks" and namespace "DebugHook". It's worth mentioning that the display name must be given, and that the rest is taken care of.
In my case, I never imagined I could put a name with a whitespace.